docs/reference/modules/indices/recovery.asciidoc

```diff
@@ -1,33 +1,39 @@
 [[recovery]]
 === Indices Recovery
 
-<<cat-recovery,Peer recovery>> is the process used to build a new copy of a
-shard on a node by copying data from the primary. {es} uses this peer recovery
-process to rebuild shard copies that were lost if a node has failed, and uses
-the same process when migrating a shard copy between nodes to rebalance the
-cluster or to honor any changes to the <<modules-cluster,shard allocation
-settings>>.
+<<cat-recovery, Peer recovery>> syncs data from a primary shard to a new or
+existing shard copy.
 
-The following _expert_ setting can be set to manage the resources consumed by
-peer recoveries:
+Peer recovery automatically occurs when {es}:
+
+* Recreates a shard lost during node failure
+* Relocates a shard to another node due to a cluster rebalance or changes to the
+<<modules-cluster, shard allocation settings>>
+* Restores a <<modules-snapshots, snapshot>>
+
+[float]
+==== Peer recovery settings
+You can use the following _expert_ settings to manage resources for peer
```

Comment:
The first setting (`indices.recovery.max_bytes_per_sec`) is not an expert-level setting and is quite often adjusted. Let's separate it from the second one which should be attributed as expert-level and will very rarely need adjustment.